images/node/aos-node-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Should be in `images/node/scripts/` if these will *only* ever be run as part of the node, otherwise they should be in `contrib/node/bin`